cloudferrylib/os/network/neutron.py

```diff
@@ -557,10 +557,12 @@ def get_networks_list(self, tenant_id=''):
         return self.neutron_client.list_networks(
             tenant_id=tenant_id)['networks']
 
+    def get_subnets_list(self, tenant_id=''):
```

Comment:
No, `None` will break current functionality. We use `tenant_id=''` by default, because in this case there will be ALL subnets returned (for all tenants).